homeassistant/components/velbus/__init__.py

```diff
@@ -167,40 +169,40 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 class VelbusEntity(Entity):
     """Representation of a Velbus entity."""
 
-    def __init__(self, channel):
+    def __init__(self, channel: VelbusChannel) -> None:
         """Initialize a Velbus entity."""
         self._channel = channel
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Get unique ID."""
-        if (serial := self._channel.get_module_serial()) == 0:
-            serial = self._channel.get_module_address()
+        if (serial := self._channel.get_module_serial()) == "":
+            serial = str(self._channel.get_module_address())
```

Comment:
```suggestion
        serial = self._channel.get_module_serial() or str(self._channel.get_module_address())
```
Maybe this needs a line break. 

You could consider using `_attr_unique_id` and `_attr_name` etc. maybe in a follow up PR

Reply:
the _attr changes are locally ready, but they depend on this PR so i can not push it :)

Reply:
This suggestion would be a better fit for the follow up then 👍